.pullapprove.yml

```diff
@@ -0,0 +1,56 @@
+# You *must* specify "version: 2"
+version: 2
+
+requirements:
+  signed_off_by:
+    required: false
+
+always_pending:  # A pull request matching any of these conditions will always have a "pending" status for this group.
+  title_regex: '(WIP|wip)'
+  explanation: 'Work in progress...'
+
+# Group settings to apply to all groups by default, optionally being overridden later
+group_defaults:
+  reset_on_push: # Whether or not approval statuses get reset to "pending" when new commits are pushed to a PR
+    enabled: false # Set to false for efficiency
+  reset_on_reopened: # Whether or not approval statuses get reset to "pending" when a pull request is re-opened
+    enabled: true
+  author_approval: # Whether or not the author must approve the pull request.
+    required: false
+    ignored: true
+  conditions:
+    branches:
+      - master  # Only require reviews when PR is merging into master
```

Comment:
This situation hasn't really come up yet, but I'd rather err on the side of being too strict for now. We can always open it up later if needed.